polling_stations/apps/data_importers/management/commands/teardown.py

```diff
@@ -0,0 +1,89 @@
+from django.apps import apps
+from django.core.management.base import BaseCommand
+
+from addressbase.models import UprnToCouncil
+from councils.models import Council
+from data_importers.models import DataQuality
+from pollingstations.models import PollingStation, PollingDistrict
+
+"""
+Clear PollingDistrict and PollingStation models
+Clear polling_station_id field in UprnToCouncil model
+Clear report, num_addresses, num_districts and num_stations
+fields in DataQuality model
+"""
+
+
+class Command(BaseCommand):
+
+    """
+    Turn off auto system check for all apps
+    We will maunally run system checks only for the
+    'data_collection' and 'pollingstations' apps
```

Comment:
While I was looking for references to `data_collection` in the migrations I realised there are some others I missed. The other files which still reference the `data_collection` app in some way are:

- `README.md`
- `pytest.ini`
- `polling_stations/apps/data_importers/management/commands/import.py`
- `polling_stations/apps/data_importers/management/commands/data_quality_report.py`
- `polling_stations/apps/data_importers/management/commands/teardown.py`

Can you have a look through those and update/bin them as necessary. I think a lot of them are actually just comments and stuff